### DISCUSSION TITLE
Could not update to latest version with docker-compose

### DISCUSSION THREAD

**Discussion by rukeba** [outline/outline#1614] [General] [answered]
Hi,

First thanks for great software!

I have self hosted dockerized installation.
Yesterday i have update outline image in docker to latest version (i think it is 0.49)

```
sudo docker-compose images
              Container                      Repository         Tag       Image Id       Size
-----------------------------------------------------------------------------------------------
outline-wiki-docker-compose_certbot_1    certbot/certbot       latest   0091bcdbf8c6   92.31 MB
outline-wiki-docker-compose_minio_1      minio/minio           latest   4a1bbd112c0c   59 MB
outline-wiki-docker-compose_nginx_1      nginx                 latest   f35646e83998   132.9 MB
outline-wiki-docker-compose_outline_1    outlinewiki/outline   latest   4ce60f3ea4f2   1.127 GB
outline-wiki-docker-compose_postgres_1   postgres              12       492fb9ae4e7a   313.9 MB
outline-wiki-docker-compose_redis_1      redis                 latest   62f1d3402b78   104.2 MB
```

I did `up` and 'restart' a few times. But when i open outline settings, i see:

```
INSTALLATION
v0.48.1
1 version behind
```

![image](https://user-images.githubusercontent.com/127363/97677394-dd004400-1a9a-11eb-9cce-78068d0ffa9c.png)

What is my mistake?

**Comment by MDXDave** [comment, marked as answer]
Its only a visual issue, you have installed version v0.49 but @tommoor tagged/built the version before he updated the `package.json`, thats why its still shown as 0.48.1: https://github.com/outline/outline/compare/v0.49.0...develop :)

**Reply by tommoor** [reply]
Ah I did publish the release notes from GitHub and then pushed the tag, you’re right. I didn’t realise it had this affect in the moment.

Thanks for bringing it to attention

**Reply by rukeba** [reply]
Thank you.
